### PR TITLE
networkd/udev: clean up parsing and handling of MTU values

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: src/network/netdev/netdev-gperf.gperf [hunk 1]
```diff
@@ -40,7 +40,7 @@ Match.Architecture,                config_parse_net_condition,           CONDITI
 NetDev.Description,                config_parse_string,                  0,                             offsetof(NetDev, description)
 NetDev.Name,                       config_parse_ifname,                  0,                             offsetof(NetDev, ifname)
 NetDev.Kind,                       config_parse_netdev_kind,             0,                             offsetof(NetDev, kind)
-NetDev.MTUBytes,                   config_parse_iec_size,                0,                             offsetof(NetDev, mtu)
+NetDev.MTUBytes,                   config_parse_mtu,                     0,                             offsetof(NetDev, mtu)
```
Comment (yuwata) [inline src/network/netdev/netdev-gperf.gperf]: I think `AF_UNSPEC` should be used here instead of `0` for consistency to other places.

Comment (poettering) [reply src/network/netdev/netdev-gperf.gperf]: true!

**Comment by poettering** [comment]
Force pushed a new version with the suggested fix. PTAL!

**Comment by poettering** [comment]
```
KVM: entry failed, hardware error 0x0
EAX=00000000 EBX=00000000 ECX=00000000 EDX=000306a9
ESI=00000000 EDI=00000000 EBP=00000000 ESP=00000000
EIP=0000fff0 EFL=00000002 [-------] CPL=0 II=0 A20=1 SMM=0 HLT=0
```
CI failure appears unrelated. Merging

**Pull request merged**